src/index.ts

```diff
@@ -0,0 +1,518 @@
+import translator from 'google-translate-api-x';
+import { writeFileSync, existsSync, readFileSync } from 'fs';
+import * as yaml from 'js-yaml';
+
+const settings = {
+	langPaths: ['./src/locales/'],
+	rootLangFiles: ['./src/locales/en.yml'],
+	rootLangCode: 'en',
+	outputLangs: [
+		'en',
+		'id',
+		'da',
+		'de',
+		'es',
+		'fr',
+		'hr',
+		'it',
+		'lt',
+		'hu',
+		'nl',
+		'no',
+		'pl',
+		'pt',
+		'ro',
+		'fi',
+		'sv',
+		'vi',
+		'tr',
+		'cs',
+		'el',
+		'bg',
+		'ru',
+		'uk',
+		'hi',
+		'th',
+		['zh-CN', 'zh'],
+		'ja',
+		'ko',
+	],
+	translateDelay: 100,
+	debug: true,
+};
+
+const colors = {
+	blue: (str: string): string => `\x1B[1;34m${str}\x1B[1;0m`,
+	green: (str: string): string => `\x1B[1;32m${str}\x1B[1;0m`,
+	magenta: (str: string): string => `\x1B[1;35m${str}\x1B[1;0m`,
+};
+
+const debug = (text: string): void => {
+	if (settings.debug) return console.log(text);
+};
+
+debug(
+	`Welcome to the ${colors.green(
+		'Interchat Translation Service',
+	)} - Made by tnfAngel and LautyDev`,
+);
+
+const translatorError = (text: string): void => {
+	throw new Error(` ${text}`);
+};
+
+const wait = (time: number) => new Promise((res) => setTimeout(res, time));
+
+const getArrayKey = (array: string | string[]) =>
+	Array.isArray(array) ? array[0] : array;
+
+const getArrayValue = (array: string | string[]) =>
+	Array.isArray(array) ? array[1] : array;
+
+const writeYamlFileSync = (filePath: string, content: Record<string, any>) => {
+	const yamlContent = yaml.dump(content);
```

Comment:
```suggestion
	const yamlContent = yaml.dump(content, { lineWidth: Number.MAX_SAFE_INTEGER });
```
Its better to remove these new lines created by line width limits. Makes embeds look bulky